spine-c/include/spine/Animation.h

```diff
@@ -60,6 +60,8 @@ typedef struct spAnimation {
 } spAnimation;
 
 spAnimation* spAnimation_create (const char* name, int timelinesCount);
+spAnimation* spAnimation_createWithTimelines (const char* name, float duration,
```

Comment:
But we can trick a bit: we can call `spAnimation_create` with `timelinesCount` equals to 0 and set `timelines` and `timelinesCount` afterwards :smile: